### PR TITLE
HDDS-9977. Dashboard for create key metrics

### PR DESCRIPTION
**Pull request by muskan1012**
## What changes were proposed in this pull request?
Created a new dashboard and added visualisations for all the required CreateKey Metrics.

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-9977

## How was this patch tested?

The patch has been tested over a Grafana.
image

### CODE REVIEW

**Review by tanvipenumudy — COMMENTED** [review]
Thank you @muskan1012, for working on the patch. 

Could you please share snapshots to the added dashboards over the PR?
Here are a few references: [Dashboard for Chunks & Volume IO](https://github.com/apache/ozone/pull/6554), [S3 API-level Dashboard](https://github.com/apache/ozone/pull/6202)

**Comment by tanvipenumudy** [comment]
@devabhishekpal could you also please take a look? Thanks!

**Comment by muskan1012** [comment]
Sure @tanvipenumudy, please find snapshot for the dashboards here:
image
image

**Review by tanvipenumudy — CHANGES_REQUESTED** [review]
I have tested out the patch locally by setting up an Ozone docker-compose cluster with the monitoring add-ons, steps as outlined [here](https://github.com/apache/ozone/blob/master/hadoop-ozone/dist/src/main/compose/ozone/README.md#add-ons).

I noticed that a few charts weren't coming up for me. It appears there are `31` occurrences of hardcoded host values. Could you please fix the same?

Additionally, once the updates are made, could you also please ensure the snapshots (as to how the patch was tested) include all the charts added for the create keys dashboard?

File: hadoop-ozone/dist/src/main/compose/common/grafana/dashboards/Ozone - CreateKey Metrics.json [hunk 1]
```diff
@@ -0,0 +1,3983 @@
...
+              "range": true,
+              "refId": "A",
+              "useBackend": false
+            }
+          ],
+          "title": "Check access latency (ops per sec)",
+          "type": "timeseries"
+        }
+      ],
+      "title": "Om Check Access Latency Metrics",
```
Comment (tanvipenumudy) [inline hadoop-ozone/dist/src/main/compose/common/grafana/dashboards/Ozone - CreateKey Metrics.json:255]:
```suggestion
      "title": "OM Check Access Latency Metrics",
```

File: hadoop-ozone/dist/src/main/compose/common/grafana/dashboards/Ozone - CreateKey Metrics.json [hunk 2]
```diff
@@ -0,0 +1,3983 @@
...
+              "range": true,
+              "refId": "A",
+              "useBackend": false
+            }
+          ],
+          "title": "Ugi Metrics (no. of ops)",
+          "type": "timeseries"
+        }
+      ],
+      "title": "Om Ugi Metrics",
```
Comment (tanvipenumudy) [inline hadoop-ozone/dist/src/main/compose/common/grafana/dashboards/Ozone - CreateKey Metrics.json:2077]:
```suggestion
      "title": "OM Ugi Metrics",
```

**Review by tanvipenumudy — COMMENTED** [review]
Please find a few minor nits.

**Review by tanvipenumudy — APPROVED** [review]

**Comment by tanvipenumudy** [comment]
Thank you @muskan1012 for working on the create keys dashboard, the changes LGTM.

This should be very helpful in diagnosing performance-related issues/regressions.

**Pull request merged**